### ISSUE TITLE
Adding support for Evidence Level Statements to the v0 Core IM

### ISSUE THREAD

**Issue by mbrush** [ga4gh/va-spec#76]
ClinGen and VICC teams are developing their initial iteration of an Implementation profile of the VA Core IM for their metakb-cvc use case. I am encouraged to see that there is very good structural alignment  so far- if we frame their model as a specialization of the 'Evidence-Level Statement' modeling paradigm.

This paradigm was not initially going to be included in the v0 release, but I have added it to the model in order to cover the metakb-cvc use case.  It provides a modeling a pattern for statements that describe the overall level/strength of evidence asserted to exist for or against a particular proposition/possible fact (e.g. in your case, the pathogenicity of a variation) . . .  and lets you assign a final classification of the variant based on this assessment (e.g. B, LB, VUS, LP, P)
 
I added support for this paradigm to the VA Core IM schema spreadsheet, and some rough documentation about the Evidence Level Statement paradigm to the VA-Spec doc [here](https://docs.google.com/document/d/18vBMg6nl7S1rAUCSoybo8ONeNGK5ObuV2WhDhZJBmLg/edit?pli=1#heading=h.32n5gqonzc4r).  
 
I would love to get some feedback about this at some point - Thanks!

**Comment by mbrush** [comment]
**Update:** 
After further discussion, we may need this model to support assessment of the overall **evidence strength** for/against a target proposition, AND/OR the agent's **confidence** that the target proposition is true or false.   This could be achieved by broadening the Evidence Level Statement object to accommodate both types of assessments, and renaming it  something like '*Proposition Assessment Statement*'.  

Implementing this in the model simply requires adding a few fields related to confidence assessment to the Statement definition (e.g. `confidence_direction`, `confidence_level`).  I have done so in the latest iteration of the [VA model spec](https://docs.google.com/spreadsheets/d/19Rp5DLFS3a1Yvvw84mPupgEkCMFRKax4LXCcwX84Vbg/edit#gid=281905833&range=219:219) and [documentation](https://docs.google.com/document/d/18vBMg6nl7S1rAUCSoybo8ONeNGK5ObuV2WhDhZJBmLg/edit?pli=1#heading=h.4n73i6edie6q) - so we can assess how this aligns with / supports the metakb-cvc use case.

**Comment by ahwagner** [comment]
FWIW I agree with the notion of supporting these as separate properties and we are doing so in our current dev commit ([evidence attributes](https://github.com/ga4gh/va-spec/blob/c7517b4796aacdd328865802efcd9de1982b0a5f/schema/annotation-source.yaml#L141-L149); [confidence attributes](https://github.com/ga4gh/va-spec/blob/c7517b4796aacdd328865802efcd9de1982b0a5f/schema/annotation-source.yaml#L29-L38)). The differences appear minor: our current commit does not use the term `evidence_strength` but instead `evidence_level`, and we put `evidence_level` and `evidence_score` in the generic `statement` and do not construct a new statement class for it.

Seems like our thoughts are aligned here though.

**Comment by mbrush** [comment]
Yes, looks like things are aligning well!  Heads up Alex that you were too quick on the uptake in looking at the model spreadsheet - and I hadn't properly completed the definitions of the updated confidence level and direction attributes when you reviewed.  They should make much more sense now! 

One thing to consider for your model is dedicated fields for confidence_direction vs evidence_direction, as seen in the VA model.  Being that confidence level and evidence level are separate things, it makes sense to be able to separately describe their respective direction.
